config/socket/socket.js

```diff
@@ -1,60 +1,60 @@
-const Game = require('./game');
-const Player = require('./player');
-require('console-stamp')(console, 'm/dd HH:MM:ss');
-const mongoose = require('mongoose');
+var Game = require('./game');
```

Comment:
There is no point of changing this file. Please revert back to how it is on staging, which is using ES6 syntax